README.md

```diff
@@ -32,6 +32,8 @@ AudienceStream™ allows you to create a unified view of your customers, correla
 
 ## Change Log
 
+- 2.0.0 Rewrite to support ROPM and updated sample app to use SceneGraph components
```

Comment:
we probably need to add in the changelog the fact that we changed the event endpoint and the fact that we are now sending a post